spec/features/features_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'takeaway'
+require 'menu'
+
+  describe 'Feature Tests' do
+
+    it 'allows the customer to view the available dishes with prices' do
+      # As a customer
+      # So that I can check if I want to order something
+      # I would like to see a list of dishes with prices
+      t = Takeaway.new
+      expect(t.show_menu).to include("Mixed Kebab")
+    end
+
+    it 'allows the customer to select from the available dishes' do
+      # As a customer
+      # So that I can order the meal I want
+      # I would like to be able to select some number of several available dishes
+      t = Takeaway.new
+      t.add("Mixed Kebab")
+      expect(t.order.current_order).to include("Mixed Kebab")
+    end
+
+    it 'allows the customer to view their order and total price' do
+      #As a customer
+      # So that I can verify that my order is correct
+      # I would like to check that the total I have been given matches the sum of the various dishes in my order
```

Comment:
Metrics/LineLength: Line is too long. [112/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)